From aa97b53fcc221332520285b62fc9b7a177d7d850 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Thu, 31 Oct 2019 17:04:29 +0800
Subject: [PATCH] drm/drv: fix drm_device_get_by_name get drm failed

the new drm driver remove idr alloc for DRM_MINOR_CONTROL, so we use
DRM_MINOR_PRIMARY to get drm device for dmc.

Change-Id: I5ac524cb98bfc4431305e341b5c659aa865bb670
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 drivers/gpu/drm/drm_drv.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/drm_drv.c b/drivers/gpu/drm/drm_drv.c
index 7f978cc64c5e..5d1dca85f994 100644
--- a/drivers/gpu/drm/drm_drv.c
+++ b/drivers/gpu/drm/drm_drv.c
@@ -268,7 +268,7 @@ struct drm_device *drm_device_get_by_name(const char *name)
 	for (i = 0; i < 64; i++) {
 		struct drm_minor *minor;
 
-		minor = drm_minor_acquire(i + DRM_MINOR_CONTROL);
+		minor = drm_minor_acquire(i + DRM_MINOR_PRIMARY);
 		if (IS_ERR(minor))
 			continue;
 		if (!minor->dev || !minor->dev->driver ||
-- 
2.35.3

